### PR TITLE
Content

### PR DESCRIPTION
**Pull request by LasseNordahl**

### CODE REVIEW

**Comment by LasseNordahl** [comment]
content + sponsors page link

File: app/client/src/assets/data/faq-questions.js [hunk 1]
```diff
@@ -1,60 +1,74 @@
...
     question:
       "Am I allowed to come with a project pre-built or will I be building the project from scratch?",
     answer:
-      "You will be building projects from scratch, although we will be offering starter packs to help you get started. You are encouraged to come with ideas of what you want to build."
+      "You will be building projects from scratch, although we will be offering starter packs to help you get started if you are a beginner hacker. You are encouraged to come with ideas of what you want to build."
+  },
+  {
+    question: "I have another question!",
+    answer:
+      "Please email us at hackuci@gmail.com if you have any additional questions! We look forward to hearing from you."
```
Comment (zpinto) [inline app/client/src/assets/data/faq-questions.js:60]: can you make the email a mailto?

**Review by zpinto — CHANGES_REQUESTED** [review]

**Review by zpinto — COMMENTED** [review]
Can you also change the dates to Jan 29-31 for hackathon and apply before Jan 24

**Review by zpinto — COMMENTED** [review]
Can you add a dollar sign?

Screen Shot 2020-12-07 at 10 45 23 PM

**Review by zpinto — APPROVED** [review]

**Pull request merged**